persistence/repository/Paginator_test.go

```diff
@@ -0,0 +1,35 @@
+package repository_test
+
+import (
+	"testing"
+
+	"github.com/ditrit/badaas/persistence/repository"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test(t *testing.T) {
+	testCases := []struct {
+		desc  string
+		page  uint
+		count uint
+	}{
+		{
+			desc:  "page=0",
+			page:  0,
+			count: 12,
+		},
+		{
+			desc:  "page=2",
+			page:  2,
+			count: 12,
+		},
+	}
+	for _, tC := range testCases {
```

Comment:
Like I say before, we don't want to have complexity in unit test, remove this loop usage.